modules/tinymce/src/core/main/ts/api/dom/ControlSelection.ts

```diff
@@ -272,7 +273,7 @@ const ControlSelection = (selection: EditorSelection, editor: Editor): ControlSe
     editor.nodeChanged();
   };
 
-  const showResizeRect = (targetElm: Element) => {
+  const showResizeRect = (targetElm: HTMLElement) => {
```

Comment:
The type here was incorrect, as it was actually a `HTMLElement`, so I've fixed that as well.